### ISSUE TITLE
Exception from enchant library

### ISSUE THREAD

**Issue by exploide** [AtomLinter/linter-pylint#106]
Sometimes when linting is triggered, an uncaught exception is displayed in Atom.

```
Error: Exception ignored in: > Traceback (most recent call last): File "/usr/lib/python3.4/site-packages/enchant/init.py", line 244, in del File "/usr/lib/python3.4/site-packages/enchant/init.py", line 273, in _free TypeError: 'NoneType' object is not callable
Error: Exception ignored in: <bound method Broker.__del__ of <enchant.Broker object at 0x7f088827cac8>>
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/enchant/__init__.py", line 244, in __del__
  File "/usr/lib/python3.4/site-packages/enchant/__init__.py", line 273, in _free
TypeError: 'NoneType' object is not callable
    at /home/jannik/.atom/packages/linter-pylint/lib/main.coffee:99:23
```

I'm not quite sure if it is a bug in Atom's linter-pylint or in enchant or something else. Let me know if you need additional information.

```
$ pylint --version
pylint 1.5.2, 
astroid 1.4.2
Python 3.4.3 (default, Jun 29 2015, 12:16:01) 
[GCC 5.1.1 20150618 (Red Hat 5.1.1-4)]
```
linter-pylint: 1.2.0
python3-enchant: 1.6.6

**Comment by Arcanemagus** [comment]
Definitely looks like a bug in `pylint` (or a plugin). When you see that pop up can you save the file at that state and try `pylint` in a terminal?

**Comment by exploide** [comment]
Indeed, today I saw the exception for the first time when running pylint from terminal.

```
Exception ignored in: <bound method Broker.__del__ of <enchant.Broker object at 0x7fd5ca4c9978>>
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/enchant/__init__.py", line 244, in __del__
  File "/usr/lib/python3.4/site-packages/enchant/__init__.py", line 273, in _free
TypeError: 'NoneType' object is not callable
```

But it is not clearly reproducible. If I lint the unchanged file a second time, it does not throw the error.

So it seems the mistake is not on linter-pylint's side. Should I close the issue?

**Comment by Arcanemagus** [comment]
Yep, closing it for you :wink:

You might want to file an issue over on [pylint](https://github.com/PyCQA/pylint/issues) so they can look into this and hopefully fix it at the source.
